cli.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
Exists to support the old way of running pacu. Unfortunately we can't name this pacu.py because various tools get confused when this is named the same as the module.

I believe people will figure this out, or worst case look at the README.md (which then suggests installing through pip) when it doesn't exist anymore. But, just thinking now, maybe we could maybe add a note to the top of the README.md as well.